external/@wordpress/blocks/index.js

```diff
@@ -19,14 +28,20 @@ export function registerBlock( namespace, block, settings ) {
  * @return {?Object}           Block settings
  */
 export function getBlockSettings( namespace, block ) {
+	if ( ! blocks[ namespace ] || ! blocks[ namespace ][ block ] ) {
+		return null;
+	}
 
+	return blocks[ namespace ][ block ];
 }
 
 /**
  * Returns all registered blocks.
  *
- * @return {Object} Block settings keyed by block name
+ * @return {Array} Blocks settings
```

Comment:
From my experience with the `multi-instance` prototype. This function was used in two places:

 - Fetching all the blocks to show in the block inserter (Array is better here, because we need to group them by category and not by namespace)

 - Fetching all the blocks to find the blocks that a block can be switched to (The filtering happens using one of the settings key and not the namespace).

So IMO it's better to return a simple array.

Regarding the name, I'm leaning towards removing the namespace and encouraging a convention